src/experimental/react/controllers/map-controller.js

```diff
@@ -0,0 +1,271 @@
+/* global setInterval, clearInterval */
```

Comment:
Let's call this `animated-map-controller.js` / `AnimatedMapController` for now.

Then you can also export it as an additional experimental export at the end of `src/index.js` so that I can use it.